<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width initial-scale=1" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Home</title>
    <meta name="description" content="">

    <link rel="stylesheet" href="https://b2du.github.io//css/main.css">
    <link rel="canonical" href="https://b2du.github.io//">
    <!-- <link rel="stylesheet" href="/css/more.css"> -->
    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
      ga('create', 'UA-118149906-1', 'auto');
      ga('send', 'pageview');
    </script>
</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="//">Bianca Dumitrascu</a>

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242"
          d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0
          h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242"
          d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484
          h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242"
          d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0
          c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
	
        
          
          <a class="page-link" href="/">Home</a>
          
        
      </div>
    </nav>

  </div>

</header>


    <div class="page-content">
      <div class="wrapper">
        <p><img src="photoBD.jpg" alt="Me" style="width: 360px; border-radius: 15px" /></p>

<h2 id="news">News!</h2>


<p>
  I lead the <a href="https://computational-morphogenomics-group.github.io/"> Computational Morphogenomics Group. We are moving to Columbia University Spring 2023, and we have many opportunities for interns, students, and postdocs. Get in touch if interested!
  
</p>

<h2 id="about-me">About me</h2>

<p> 
	I am an Affiliated Lecturer in the <a href="https://www.cst.cam.ac.uk/"> Department of Computer Science and Technology </a> (Computer Laboratory) at the University of Cambridge as a Departmental Early Career Fellow in the Accelerate Programme for Scientific Discovery. If you are a prospective graduate student interested in joining my team or a researcher interested in collaborating contact me via email at <em>bmd39@cam.ac.uk</em>.
 
	I work at the intersection of machine learning and genetics. My main research interest is understanding how local molecular rules give raise to emergent spatial patterns in the context of biological dynamical systems. To this end, I use techniques from statistical optimization, statistical physics and domain adaptation to identify contextual phenotypes in spatial transcriptomic data and to understand the identity of single cells and their interactions in early developement. I am also interested in active learning and graphical neural networks as models to study the effects and side-effects of drug cocktails.
</p>


<p> Previously, I was a Member in the School of Mathematics at the Institute for Advanced Study 
and I attended the semester long program in <a href="https://www.samsi.info/programs-and-activities/semester-long-programs/2019-fall-semester-program-on-deep-learning/">deep learning</a> at the Statistical and Applied Mathematical Sciences Institute, where I was graciously hosted by <a href="https://stat.duke.edu/people/david-b-dunson">David Dunson</a> in the Duke Statistical Science Department. I received my PhD in Computational Biology at Princeton University</a>, 
under the mentorship of <a href="https://www.cs.princeton.edu/~bee/">Barbara E. Engelhardt</a>.
My PhD research focused on the effect of experimental design in single cell gene expression studies and on method development for structured, high-dimensional medical and genomic data. I did my undergraduate studies in Mathematics at MIT.



<h2 id="publications"> Peer-Reviewed Publications</h2>
[* equal contribution]
[&#9830 corresponding author]
	    
<p> &#9749 <em> Deep learning for bioimage analysis in developmental biology
 </em> <br/> 
Adrien Hallou*, Hannah G. Yevick*, &#9830 <b>Bianca Dumitrascu</b>, &#9830 Virginie Uhlmann<br/> 
Development 15 September 2021; 148 (18): dev199616 <a href="https://doi.org/10.1242/dev.199616">[paper]</a>
</p>
	    
<p> &#9749 <em> Optimal gene selection for cell type discrimination in single cell analyses
 </em> <br/> 
<b>Bianca Dumitrascu*</b>, Soledad Villar*, Dustin G Mixon, Barbara E Engelhardt<br/>
Nature communications 12 (1), 1-8, 2021 <a href="https://doi.org/10.1038/s41467-021-21453-4">[paper]</a>
</p>
	    
<p> &#9749 <em> Causal Network Inference from Gene Transcriptional Time Series Response to Glucocorticoids
 </em> <br/> 
Jonathan Lu*, <b>Bianca Dumitrascu*</b>, Ian C McDowell, Brian Jo, Alejandro Barrera, Linda K. Hong,Sarah M. Leichter, Timothy E. Reddy, Barbara E. Engelhardt<br/> 
PLOS Computational Biology, 2021 <a href="https://www.biorxiv.org/content/10.1101/587170v1">[bioRxiv paper]</a>
</p>


<p>  &#9749 <em>  Sparse multi-output Gaussian processes for online medical time series prediction
</em> <br/> 
Li-Fang Cheng, <b>Bianca Dumitrascu</b>, Gregory Darnell, Corey Chivers, Michael E Draugelis, Kai Li, and Barbara E Engelhardt <br/> 
BMC Medical Informatics and Decision Making 20.1 (2020): 1-23
</p>

<p> &#9749 <em> Nonparametric Bayesian multi-armed bandits for single cell experiment design </em> <br/> 
Federico Camerlenghi*, <b>Bianca Dumitrascu</b>*, Federico Ferrari*, Barbara E. Engelhardt, Stefano Favaro <br/>
Annals of Applied Statistics, 2020 <a href="https://projecteuclid.org/euclid.aoas/1608346909">[paper]</a>
</p>


<p > &#9749 <em>  Patient-Specific Effects of Medication Using Latent Force Models with Gaussian Processes </em> <br/>
Li-Fang Cheng, <b>Bianca Dumitrascu</b>, Michael Zhang, Corey Chivers, Michael Draugelis, Kai Li, Barbara E Engelhardt </br>
AISTATS 2020 <a href="https://arxiv.org/abs/1906.00226"> [arXiv paper] </a>
</p>


<p > &#9749 <em> netNMF-sc: leveraging geneâ€“gene interactions for imputation and dimensionality reduction insingle-cell expression analysis </em> <br/>
Rebecca Elyanow, <b>Bianca Dumitrascu</b>, Barbara E Engelhardt, and Benjamin J Raphael <br/> 
Genome Research 30: 195-20, 2020; RECOMB 2019 <a href="https://genome.cshlp.org/content/early/2020/01/28/gr.251603.119">[paper]</a>
</p>

<p > &#9749 <em> End-to-end training of deep probabilistic CCA for joint modeling of paired biomedical observations </em> <br/>
Gregory Gundersen, <b>Bianca Dumitrascu</b> , Jordan T. Ash, Barbara E. Engelhardt <br/>
UAI 2019 <a href="http://proceedings.mlr.press/v115/gundersen20a.html">[paper]</a>
</p>

<p>  &#9749 <em> PG-TS: Improved Thompson Sampling for Logistic Contextual Bandits</em> <br/>
<b>Bianca Dumitrascu</b>*, Karen Feng*, and Barbara E Engelhardt <br/> 
NeurIPS 2018 <a href="https://papers.nips.cc/paper/7713-pg-ts-improved-thompson-sampling-for-logistic-contextual-bandits.pdf"> [paper] </a>
</p>

 <p> &#9749 <em> Statistical tests for detecting variance effects in quantitative trait studies</em><br/> 
<b>Bianca Dumitrascu</b>, Gregory Darnell, Julien Ayroles, and Barbara E Engelhardt <br/> 
Bioinformatics, 2018 <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty565/5050024">[paper]</a>
</p>

<p> &#9749 <em> BIISQ: Bayesian nonparametric discovery of Isoforms and Individual Specific Quantification from RNA-seq data
</em><br/> 
Derek Aguiar, Li-Fang Cheng, <b>Bianca Dumitrascu</b>, Fantine Mordelet, Athma Pai, and Barbara E Engelhardt <br/> 
Nature Communications, 9(1), 2018  <a href="https://www.nature.com/articles/s41467-018-03402-w">[paper]</a>
</p>


<h2 id="prepublications">Preprints</h2>


<p> &#9749 <em> Sequential Gaussian Processes for Online Learning of Nonstationary Functions </em> <br/> 
Michael Minyi Zhang, <b>Bianca Dumitrascu</b>, Sinead A. Williamson, Barbara E. Engelhardt <a href="https://arxiv.org/abs/1905.10003">[arXiv paper]</a>
</p>


<p> &#9749 <em> GT-TS: Experimental design for maximizing cell type discovery in single-cell data</em><br/> 
<b>Bianca Dumitrascu</b>, Karen Feng, and Barbara E Engelhardt 
<a href="https://www.biorxiv.org/content/early/2018/08/07/386540">[bioRxiv paper]</a>
</p>

<p> &#9749 <em> Sparse Multi-Output Gaussian Processes for Medical Time Series Prediction </em><br/>
Li-Fang Cheng, Gregory Darnell, <b>Bianca Dumitrascu</b>, Corey Chivers, Michael E Draugelis, Kai Li, and Barbara E Engelhardt <a href="https://arxiv.org/abs/1703.09112"> [arXiv paper]</a>
</p>




<!---<h2 id="publications">Publications</h2>
<p> <a href = "https://scholar.google.com/citations?user=fn0yg8cAAAAJ&hl=en"> Google Scholar</a>
</p>
-->

<!--

<h2 id="talks">Invited Talks</h2>
<ul>
  <li><strong> Bandits and Experimental Design </strong> <br/>
    Models, Inference, and Algorithms, Broad Institute of MIT and Harvard, 2018<br/>
    <a href="https://www.youtube.com/watch?v=sDFe3e33m_I">video</a>
  </li>

  <li><strong> Exploring the Glucocorticoid Receptor network: challenges in causal inference </strong> <br/>
    Probabilistic Modeling in Genomics, Oxford University, 2016<br/>
  Joint work with J. Lu, B. Jo, I. C. McDowell, T. Reddy, and B. E. Engelhardt
  </li>

  <li><strong> Detection of variance controlling quantitative traits loci </strong>  <br/>
    New York Area Population Genomics Workshop, NYC, 2015<br/>
  Joint work with G. Darnell</a>, J. Ayroles, B. E. Engelhardt
  </li>
</ul>



<h2 id="internships">Internships</h2>
<ul>
  <li><strong>Google Research Mountain View</strong>, June - August 2017<br/>
Worked on deep variational graph embeddings for video recommendation. </li>
  <li><strong>CustomMade</strong>, June - July 2013 <br/>
Worked on a collaborative filtering framework for item recommendation. </li>
</ul>

-->

<h2 id="curriculum-vitae">Curriculum Vitae</h2>
<p>My CV can be found <a href="CV-Bianca.pdf">here</a>.</p>
      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <!-- <h2 class="footer-heading"></h2> -->

      <div class="footer-col  footer-col-1">
        <p class="social-media-list">

          
            <a href="mailto:bmd39@cam.ac.uk">
              <i class="fa fa-envelope-square fa-2x"></i>
            </a>
            
            <a href="https://letterboxd.com/b2du/films/diary/">
              <i class="fa fa-film fa-2x"></i>
            </a>

            <a href="https://scholar.google.com/citations?user=fn0yg8cAAAAJ&hl=en">
              <i class="ai ai-google-scholar"></i>
            </a>

        </p>
      </div>

    <div class="footer-col-wrapper">
      <div class="footer-col  footer-col-2">
	<p> </p>
        <!-- <ul class="contact-list"> -->
        <!--   <li> Contact </li> -->
        <!--   <li><a href="mailto:biancad@ias.edu">biancad@ias.edu</a></li> -->
        <!-- </ul> -->
      </div>

      <div class="footer-col  footer-col-3">
        <p class="text"></p>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
